ethereum/client/src/tests.rs

```diff
@@ -0,0 +1,281 @@
+use alloy::sol;
+
+#[cfg(test)]
```

Comment:
It's already under `#[cfg(test)]` in `lib.rs`

Reply:
fixed